app/models/comment.rb

```diff
@@ -8,6 +11,10 @@ class Comment < ActiveRecord::Base
   before_save :set_checked
   after_commit :notify!
 
+  def is_supervisor_check?
+    commentable.is_a? Team
+  end
+
   def for_application?
```

Comment:
Should I refactor the `is_supervisor_check?` and `for_application?` check in one method that checks the commentables?